src/main/java/de/uhd/ifi/se/decision/management/jira/filtering/impl/FilterExtractorImpl.java

```diff
@@ -222,8 +221,7 @@ private boolean checkIfKnowledgeTypeMatches(DecisionKnowledgeElement element) {
 					|| filterSettings.getDocumentationLocations().size() == 1 && filterSettings
 							.getDocumentationLocations().get(0).equals(DocumentationLocation.UNKNOWN)) {
 				// Check if the Status is filtered
-				if (filterSettings.getSelectedStatus()
-						.contains(StatusPersistenceManager.getStatusForElement(element))) {
+				if (filterSettings.getSelectedStatus().contains(element.getStatus())) {

```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [These nested if statements could be combined](https://app.codacy.com/app/UHD/cures-condec-jira/pullRequest?prid=4636195)